# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/cyme/hotfix-unknown-linux.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- cyme-1.8.4/Cargo.toml.vanilla	2024-10-04 14:22:43.216666665 +0300
+++ cyme-1.8.4/Cargo.toml	2024-10-04 14:35:08.113333339 +0300
@@ -33,25 +33,14 @@
 strum_macros = "0.26"
 regex = { version = "1.10.5", optional = true }
 uuid = { version = "1.9.1", features = ["serde"] }
+udevrs = { version = "^0.3.0", optional = true }
+udevlib = { package = "udev", version = "^0.8.0", optional = true }
+#rusb = "0.9.4"
 
 [dev-dependencies]
 diff = "0.1"
 assert-json-diff = "2.0.2"
 
-[target.x86_64-unknown-linux-gnu.dependencies]
-udevrs = { version = "^0.3.0", optional = true }
-udevlib = { package = "udev", version = "^0.8.0", optional = true }
-rusb = "0.9.4"
-
-[target.arm-unknown-linux-gnueabihf.dependencies]
-udevrs = { version = "^0.3.0", optional = true }
-udevlib = { package = "udev", version = "^0.8.0", optional = true }
-rusb = "0.9.4"
-
-[target.aarch64-unknown-linux-gnu.dependencies]
-udevrs = { version = "^0.3.0", optional = true }
-udevlib = { package = "udev", version = "^0.8.0", optional = true }
-rusb = "0.9.4"
 
 [features]
 libusb = ["dep:rusb"]
